sonarqube/datadog_checks/sonarqube/api.py

```diff
@@ -0,0 +1,56 @@
+import inflection
+
+
+class Api:
```

Comment:
nit
```suggestion
class SonarqubeAPI:
```
Api sounds too generic, imho

Reply:
Also this looks like a refactor that should be done in a separate PR because this one is pretty big

Reply:
I think the refactor is attached to the new feature
In addition, the new code structure greatly simplifies testing too

Reply:
Renamed class, I agree with you